addons/website_slides/models/slide_channel.py

```diff
@@ -540,6 +551,24 @@ def _compute_action_rights(self):
                 channel.can_comment = user_karma >= channel.karma_slide_comment
                 channel.can_vote = user_karma >= channel.karma_slide_vote
 
+    @api.depends('visibility')
+    def _compute_prerequisite_channel_ids(self):
+        for channel in self:
+            if channel.prerequisite_channel_ids and channel.visibility != channel.prerequisite_channel_ids[0].visibility:
```

Comment:
is it exactly what we want (only the first visibility matter) ? (better to double check with po)

Reply:
Normally the `prerequisite_channel_ids` can only be of the same type but I changed to
`if all(c.visibility != channel.visibility for c in channel.prerequisite_channel_ids):` :smiley: